spec/features/users_spec.rb

```diff
@@ -5,15 +5,6 @@
   let(:user) { create(:user) }
   let(:profile_path) { Hyrax::Engine.routes.url_helpers.user_path(user, locale: 'en') }
 
-  context 'when clicking all users' do
-    # TODO: Move this to a view test
```

Comment:
The following view spec (`app/views/hyrax/users/index.html.erb`) already existed.